src/HTML_Renderer/org/lobobrowser/html/style/HtmlValues.java

```diff
@@ -579,7 +579,11 @@ public static int getOldSyntaxPixelSize(String spec, final int availSize, final
     try {
       if (spec.endsWith("%")) {
         return (availSize * Integer.parseInt(spec.substring(0, spec.length() - 1))) / 100;
-      } else {
+      }
+      if (spec.endsWith("px")){
+          return (int) Math.round(Double.parseDouble(spec.substring(0, spec.length() - 2)));
+      }
+      else {
```

Comment:
Minor comment: the coding style we are following in this project is to begin the `else` on the same line as `}`.